DEVELOPMENT.md

```diff
@@ -19,7 +19,7 @@ The contents of the Development Guide include:
 
 RustPython requires the following:
 
-- Rust 1.36 or higher
+- Rust latest stable version (at least 1.38.0)
```

Comment:
We could say that we require at least the stable version, without mentioning the 1.38 version, since the stable version will move forward in time. What do you think?

Reply:
That will be the most correct description. But still my rule of thumb is that people understand the version easier with specific number. Maybe ‘(e.g 1.38.0 in oct 2019)’ can be a helpful comment

Reply:
Agreed, having an explicit version number is useful.